doc/json/casebook.json

```diff
@@ -530,7 +530,10 @@
       "timestamp" : "2016-01-01T01:01:01.000Z",
       "revision" : 10,
       "abstraction_level" : "string",
-      "alternate_terms" : [ "string" ],
+      "alternate_terms" : [ {
+        "term" : "string",
+        "description" : "string"
```

Comment:
The `"description"` must be optional.  Many `"alternate_terms"` entries don't have any description text.